source/Calamari.CloudAccounts/Calamari.CloudAccounts.csproj

```diff
@@ -22,7 +22,10 @@
     <ItemGroup>
       <PackageReference Include="AWSSDK.Core" Version="3.3.104.14" />
       <PackageReference Include="AWSSDK.SecurityToken" Version="3.3.4.34" />
-      <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+      <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
```

Comment:
Net452 limits to version 10.0.3, rather than the later 13.0.3 used in other projects